### PR TITLE
Fixed config for kotlin-tests (and directory structure)

### PR DESCRIPTION
**Pull request by anskaal**
Both AndroidTests and unit-tests should work now.

### CODE REVIEW

**Review by martintoften — APPROVED** [review]
👌

File: app/build.gradle
Comment (csdodd) [inline app/build.gradle]: Minor: Can we group together all the `test` dependencies, seperately from all the other dependencies. Kind of weird that they're all mixed in together.

Comment (anskaal) [reply app/build.gradle]: Good input! I agree :+1:

File: app/build.gradle
Comment (martintoften) [inline app/build.gradle]: Why did you comment this out?

**Comment by csdodd** [comment]
There's a couple of things here.

The first two commits can be squashed into one. The second two commits can be squashed into one.
Also, there's some commented out code that should be removed: https://github.com/bakkenbaeck/android-playbook#dont-commit-commented-out-code

**Comment by csdodd** [comment]
Also, when you open a pull request -- it should be because you want to merge it.
Then if there is feedback, you fix the pull request until it is merged.

You shouldn't open a pull request, and then carry on developing in the branch. In the worst case, we end up approving something, that you then go on to change drastically. Because it's already been approved it just gets merged anyway - without proper review.

There is no size limit on how small a pull request can be. So long as the app doesn't crash spectacularly you should merge often, with as small pull requests as possible.

File: app/build.gradle
Comment (anskaal) [reply app/build.gradle]: I should have removed it instead of commenting it out. 

@martintoften I believe it adds the tests under that path as regular unit-tests, running on the JVM instead of on an Android-device.

**Comment by anskaal** [comment]
Ok, I've squashed commits and removed unused / commented code. I'm ready to merge the pull-request if you are ok with it :)

**Comment by martintoften** [comment]
Looks good to me

File: app/build.gradle
Comment (csdodd) [inline app/build.gradle:33]: Having the opening brace on a new-line isn't the style we agreed on in the style guide.

Change to `if (!sourceSet.name.startsWith("test")) {`

File: app/src/androidTest/kotlin/explorer/city/com/cityexplorer/ExampleInstrumentedTestKotlin.kt
Comment (csdodd) [inline app/src/androidTest/kotlin/explorer/city/com/cityexplorer/ExampleInstrumentedTestKotlin.kt:25]: This is deprecated and should no longer be used. More info: https://developer.android.com/reference/android/test/suitebuilder/annotation/LargeTest.html

File: app/src/androidTest/kotlin/explorer/city/com/cityexplorer/ExampleInstrumentedTestKotlin.kt
Comment (csdodd) [inline app/src/androidTest/kotlin/explorer/city/com/cityexplorer/ExampleInstrumentedTestKotlin.kt:26]: Is this annotation needed, since you've set the testrunner in build.gradle?

**Pull request closed without merging**
